spinalcordtoolbox/template.py

```diff
@@ -50,3 +50,17 @@ def get_vertebral_level_from_slice(im_vertlevel, idx_slice):
         logger.debug('Empty slice: z=%s (%s)', idx_slice, e)
         vert_level = None
     return vert_level
+
+
+def get_all_vertebral_level(im_vertlevel):
+    """
+    Find all available vertebral levels.
+    :return: list: List of all available vertebral levels.
+    """
+    data_vertlevel = im_vertlevel.data
+    vert_level = list(np.unique(data_vertlevel))
+    # Sort vertebral levels
+    vert_level.sort()
+    # Remove 0 value since not a level
+    vert_level.remove(0)
+    return vert_level
```

Comment:
There is a similar method already present in `Image` objects: `.getNonZeroCoordinates`.

So, what if we made this function into a `Image.getNonZeroValues` method? That way, the interface is somewhat consistent....

Reply:
(Side note: The `camelCase` of the existing method isn't great? But I figure it would be good to keep similar methods organized in similar ways. That way, if a future dev decides to one day refactor `.getNonZeroCoordinates`, then the similar method will be nearby, and will likely be included in the refactoring effort.)